app/models/ability.rb

```diff
@@ -86,10 +86,8 @@ def signed_in(user)
       event.users.include?(user)
     end
 
-    # cannot create an event if program does not have open cfp
-    cannot [:new, :create], Event do |event|
-      user_inclusion = event.event_users.map { |event_user| event_user.user.id }.compact.include? user.id
-      !event.program.cfp_open? || !event.new_record? || !user_inclusion
+    can [:new, :create], Event do |event|
+      event.program.cfp_open? && event.new_record?
     end
```

Comment:
> it shouldn't be possible to create an event with someone else as a submitter, we have to think about it (probably atm possible)! 

yes, it will be possible to create an event with someone else as a submitter on your branch. `can` followed by `can` rule are `ORed`. Equivalent rule for what you have used is:
```Ruby
event.users.include?(user) ||  event.program.cfp_open? && event.new_record?
```